docs/development/sync-protocol.md

```diff
@@ -248,65 +251,285 @@ sequenceDiagram
 | valid_from | YES | int | Unix timestamp of when the cert was issued |  1647447514 |
 | valid_until | YES | int | Unix timestamp of when the cert expires |  1678983513 |
 
+#### Entitlement Info
+
+If the binary is signed and has entitlements, they are stored in the Event as an EntitlementInfo object that indicates if the entitlements were filtered followed by a list of entitlements.
+
+| Key | Required | Type | Meaning | Example Value |
+|---|---|---|---|---|
+| entitlements_filtered | Y | boolean | Did Santa filter out some of the entitlements | true |
+| entitlements | Y | list of entitlements see below | See Entitlement |
+
+##### Entitlement
+
+An entitlement is stored as an object of key- Value pairs
+
+| Key | Required | Type | Meaning | Example Value |
+|---|---|---|---|---|
+| key | Y | string | The name of the entitlement | "com.apple.security.hypervisor" |
+| value | Y | string | The value of the entitlement as a string | "true" |
+
 
 #### `eventupload` Request Example Payload
 
 ```json
 {
-  "events": [{
-    "file_path": "\/Applications\/Santa.app\/Contents\/MacOS",
-    "file_bundle_version": "9999.1.1",
-    "parent_name": "launchd",
-    "logged_in_users": [
-      "markowsky"
-    ],
-    "quarantine_timestamp": 0,
-    "signing_chain": [{
-        "cn": "Apple Development: Google Development (EQHXZ8M8AV)",
-        "valid_until": 1678983513,
-        "org": "Google LLC",
-        "valid_from": 1647447514,
-        "ou": "EQHXZ8M8AV",
-        "sha256": "7ae80b9ab38af0c63a9a81765f434d9a7cd8f720eb6037ef303de39d779bc258"
-      },
-      {
-        "cn": "Apple Worldwide Developer Relations Certification Authority",
-        "valid_until": 1897776000,
-        "org": "Apple Inc.",
-        "valid_from": 1582136027,
-        "ou": "G3",
-        "sha256": "dcf21878c77f4198e4b4614f03d696d89c66c66008d4244e1b99161aac91601f"
+  "events": [
```

Comment:
I'm not sure it's useful to have a 250 line example here. We could probably still use Santa.app which has a few entitlements to get the same point across.

Reply:
I've swapped it for a smaller one.